package.json

```diff
@@ -7,16 +7,18 @@
     "url": "git+https://github.com/danawoodman/react-fontawesome.git"
   },
   "config": {
-    "entry": "src/index.js"
+    "entry": "src",
+    "output": "lib",
+    "mainFile": "src/index.js"
   },
   "bugs": {
     "url": "https://github.com/danawoodman/react-fontawesome/issues"
   },
   "main": "lib/index.js",
   "scripts": {
-    "build": "babel -o $npm_package_main $npm_package_config_entry",
+    "build": "babel $npm_package_config_entry -o $npm_package_config_output",
```

Comment:
I'm getting `Error: EISDIR: illegal operation on a directory, open 'lib'` with this. I'm able to get it to work with:

```
babel -o lib/index.js src/index.js
```

However, this fails:

```
babel -o lib src
```

Is this working for you?

Also, it fails if the `lib` directory is not present (which it won't be if it is in `.gitignore`) so it seems we would need to have `lib/.gitkeep` in the repo.